### ISSUE TITLE
Add alt text on all images

### ISSUE THREAD

**Issue by Albadylic** [fac-17/My-Body-Back#140]
All images should have alt text for accessibility

- [x] Check all images and add alt text

Doing this would be aided by checking the console errors when running npm start. The console will direct to each file where this problem occurs.
